src/avatar.jsx

```diff
@@ -97,60 +126,32 @@ const Avatar = React.createClass({
       ...other,
     } = this.props;
 
-    let styles = {
-      root: {
-        height: size,
-        width: size,
-        userSelect: 'none',
-        borderRadius: '50%',
-        display: 'inline-block',
-      },
-    };
-
-    if (src) {
-      const borderColor = this.state.muiTheme.avatar.borderColor;
+    const {
+      prepareStyles,
+    } = this.state.muiTheme;
 
-      if (borderColor) {
```

Comment:
Oh good catch, I don't know how I missed that `borderColor` was part of the condition for applying those styles. Thanks!